src/components/subject-level-with-labels/SubjectLevelWithLabels.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Box from '@mui/material/Box'
+import { SxProps } from '@mui/material'
+import Typography from '@mui/material/Typography'
+import SubjectLevelChips from '~/components/subject-level-chips/SubjectLevelChips'
+
+import palette from '~/styles/app-theme/app.pallete'
+import { spliceSx } from '~/utils/helper-functions'
+
+import { ProficiencyLevelEnum } from '~/types'
+import { styles } from '~/components/subject-level-with-labels/SubjectLevelWithLabels.styles'
+
+interface SubjectLevelChipsProps {
+  sx?: SxProps
+  proficiencyLevel: ProficiencyLevelEnum | ProficiencyLevelEnum[]
+  subject: string
+  color?: string
+}
+
+const SubjectLevelWithlabels: FC<SubjectLevelChipsProps> = ({
+  proficiencyLevel,
+  subject,
+  color = palette.success[600],
```

Comment:
do you need to import palette here? I think, 'success.600' will work

Reply:
this color doesn't need default, subjectLevelChips has default

Reply:
i agree, its reduntant. Fixed)